### PR TITLE
Support -o rhv-upload option for virt-v2v

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: xiaodwan <xiaodwan@redhat.com>

### CODE REVIEW

File: backends/v2v/cfg/convert_destination.cfg [hunk 1]
```diff
@@ -32,6 +32,8 @@ variants:
         remote_node_password = RHV_NODE_PASSWORD
         remote_user = ${remote_node_user}
         remote_pwd = ${remote_node_password}
+        ovirt_ca_file_path = "/etc/pki/ovirt-engine/ca.pem"
+        local_ca_file_path = "/tmp/rhv_upload_ca.pem"
```
Comment (chloerh) [inline backends/v2v/cfg/convert_destination.cfg]: If these paths are not constant values, they should be put as 'XXX_V2V_EXAMPLES' and be replaced when run the cases.

**Comment by chloerh** [comment]
Others are ok to me

File: backends/v2v/cfg/convert_destination.cfg (hunk 1 above)
Comment (xiaodwan) [reply backends/v2v/cfg/convert_destination.cfg]: They are constant values.

**Comment by xiaodwan** [comment]
@chloerh @chunfuwen please help to review it, thanks!

**Comment by xiaodwan** [comment]
The test result see:
https://github.com/autotest/tp-libvirt/pull/1756

File: virttest/utils_v2v.py [hunk 2]
```diff
@@ -731,24 +745,28 @@ def import_vm_to_ovirt(params, address_cache, timeout=600):
...
-                                     cluster_name,
-                                     timeout=timeout)
-        logging.info("The latest VM list: %s", vm.list())
-    except Exception as e:
-        # Try to delete the vm from export domain
-        vm.delete_from_export_domain(export_name)
-        logging.error("Import %s failed: %s", vm.name, e)
-        return False
+
+    # if output_method is None or "" or is not 'rhv_upload', treat it as
```
Comment (chunfuwen) [inline virttest/utils_v2v.py]: if / If

File: virttest/utils_v2v.py [hunk 3]
```diff
@@ -731,24 +745,28 @@ def import_vm_to_ovirt(params, address_cache, timeout=600):
...
+            vm.import_from_export_domain(export_name,
+                                         storage_name,
+                                         cluster_name,
+                                         timeout=timeout)
+            logging.info("The latest VM list: %s", vm.list())
+        except Exception as e:
+            # Try to delete the vm from export domain
+            vm.delete_from_export_domain(export_name)
+            logging.error("Import %s failed: %s", vm.name, e)
+            return False
```
Comment (chunfuwen) [inline virttest/utils_v2v.py]: Do you really need return ?

File: virttest/utils_v2v.py [hunk 4]
```diff
@@ -812,10 +830,16 @@ def check_exit_status(result, expect_error=False, error_flag='strict'):
     """
     if not expect_error:
         if result.exit_status != 0:
-            raise exceptions.TestFail(decode_to_text(result.stderr, errors=error_flag))
+            raise exceptions.TestFail(
```
Comment (chunfuwen) [inline virttest/utils_v2v.py]: change to test.fail

**Pull request merged**